athena-tables.sql

```diff
@@ -0,0 +1,85 @@
+CREATE EXTERNAL TABLE `raw_events_code`(
+`querystring` string COMMENT 'from deserializer',
+`event` string COMMENT 'from deserializer',
+`datetime` timestamp COMMENT 'from deserializer')
+PARTITIONED BY (
+`date` string,
+`hour` int)
+ROW FORMAT SERDE
+'org.apache.hive.hcatalog.data.JsonSerDe'
+WITH SERDEPROPERTIES (
+  'timestamp.formats'='yyyy-MM-dd\'T\'HH:mm:ss')
+    STORED AS INPUTFORMAT
+    'org.apache.hadoop.mapred.TextInputFormat'
+    OUTPUTFORMAT
+    'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+    LOCATION
+    's3://<CODE_BUCKET>/fastly';
+
+
+CREATE EXTERNAL TABLE `raw_events_prod`(
+`querystring` string COMMENT 'from deserializer',
+`event` string COMMENT 'from deserializer',
+`datetime` timestamp COMMENT 'from deserializer')
+PARTITIONED BY (
+`date` string,
+`hour` int)
+ROW FORMAT SERDE
+'org.apache.hive.hcatalog.data.JsonSerDe'
+WITH SERDEPROPERTIES (
+  'timestamp.formats'='yyyy-MM-dd\'T\'HH:mm:ss')
+    STORED AS INPUTFORMAT
+    'org.apache.hadoop.mapred.TextInputFormat'
+    OUTPUTFORMAT
+    'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+    LOCATION
+    's3://<PROD_BUCKET>/fastly';
+
+
+CREATE OR REPLACE VIEW notification_received_prod AS
+  SELECT
+    "dateTime" "received_timestamp",
+    "regexp_extract"("querystring", 'notificationId=([\w-]+)', 1) "notificationId",
```

Comment:
We might want to add a WHERE != unknown clause due to the bug that's been fixed, but still populating s3 with invalid events.

Reply:
I'm not sure that's a concern the table should have